test_negative/output.expected.2.7

```diff
@@ -29,26 +29,28 @@ test_negative/negative.py:86: error: Invalid index type "str" for "_ExtensionDic
-test_negative/negative.py:93: error: Incompatible types in assignment (expression has type "int", variable has type "_ExtensionFieldDescriptor[Simple1, Any]")
-test_negative/negative.py:97: error: Incompatible types in assignment (expression has type "Union[Literal[u'b_oneof_1'], Literal[u'b_oneof_2']]", variable has type "Union[Literal[u'a_oneof_1'], Literal[u'a_oneof_2'], Literal[u'outer_message_in_oneof'], Literal[u'outer_enum_in_oneof'], Literal[u'inner_enum_in_oneof']]")
-test_negative/negative.py:100: error: "Descriptor" has no attribute "Garbage"
-test_negative/negative.py:101: error: "Descriptor" has no attribute "Garbage"
-test_negative/negative.py:104: error: "EnumDescriptor" has no attribute "Garbage"
-test_negative/negative.py:107: error: "FileDescriptor" has no attribute "Garbage"
-test_negative/negative.py:114: error: "OuterEnumValue" has no attribute "FOO"
-test_negative/negative.py:118: error: Argument 1 to "Name" of "_EnumTypeWrapper" has incompatible type "int"; expected "OuterEnumValue"
-test_negative/negative.py:120: error: Argument 1 to "Name" of "_EnumTypeWrapper" has incompatible type "InnerEnumValue"; expected "OuterEnumValue"
-test_negative/negative.py:122: error: Argument 1 to "Name" of "_EnumTypeWrapper" has incompatible type "InnerEnumValue"; expected "OuterEnumValue"
-test_negative/negative.py:124: error: Argument 1 to "Name" of "_EnumTypeWrapper" has incompatible type "InnerEnumValue"; expected "OuterEnumValue"
-test_negative/negative.py:128: error: "ScalarMap[int, unicode]" has no attribute "get_or_create"
-test_negative/negative.py:130: error: No overload variant of "get" of "Mapping" matches argument type "str"
-test_negative/negative.py:130: note: Possible overload variant:
-test_negative/negative.py:130: note:     def get(self, k: int) -> Optional[unicode]
-test_negative/negative.py:130: note:     <1 more non-matching overload not shown>
-test_negative/negative.py:131: error: No overload variant of "get" of "Mapping" matches argument type "str"
-test_negative/negative.py:131: note: Possible overload variant:
-test_negative/negative.py:131: note:     def get(self, k: int) -> Optional[OuterMessage3]
-test_negative/negative.py:131: note:     <1 more non-matching overload not shown>
-test_negative/negative.py:134: error: Incompatible types in assignment (expression has type "Optional[unicode]", variable has type "int")
-test_negative/negative.py:135: error: Incompatible types in assignment (expression has type "Optional[OuterMessage3]", variable has type "int")
-Found 44 errors in 1 file (checked 19 source files)
+test_negative/negative.py:90: error: Argument 1 to "HasExtension" of "Message" has incompatible type "_ExtensionFieldDescriptor[Simple2, SeparateFileExtension]"; expected "_ExtensionFieldDescriptor[Simple1, Any]"
```

Comment:
Would be very cool if the `expected` line had a union of the possible extensions, but that seems fairly difficult - as it involves the rendering of a message to include information from all generated code - so this seems good enough!